packages/synapse-react-client/src/style/components/_query-wrapper-plot-nav.scss

```diff
@@ -2,92 +2,80 @@
 @use 'sass:map';
 
 .QueryWrapperPlotNav {
-  .SRC-wrapper {
-    display: grid;
-  }
+  display: grid;
+
   @media (min-width: map-get(SRC.$breakpoints, 'medium')) {
-    .SRC-wrapper {
-      grid-template-columns: 25% 75%;
+    grid-template-columns: 25% 75%;
 
-      // by default, children take up the full row
-      > * {
-        grid-column: 1 / span 2;
+    // by default, children take up the full row
+    > * {
+      grid-column: 1 / span 2;
+    }
+
+    // column (depends on if we are showing facet filters)
+    .isShowingFacetFilters {
+      &.FacetFilterControls {
+        grid-column: 1 / span 1;
+        grid-row: 3 / span 100;
       }
-      &.has-facets {
-        grid-template-columns: 25% 75%;
-        > * {
-          grid-column: 1 / span 2;
-        }
```

Comment:
Removed `.has-facets`; the CSS is identical +/- this class, so I don't think it was needed anymore anyways